### DISCUSSION TITLE
Freeform - Limit access to individual export profiles

### DISCUSSION THREAD

**Discussion by ellierider** [solspace/craft-freeform#971] [Feature Requests]
In Freeform 4, is there a way to limit access to individual export profiles?

We would like to make it so that an assigned user can export a particular form, but not run all export profiles.

**Comment by kjmartens** [comment]
Hi @ellierider,

Unfortunately this isn't possible yet, but we've noted it for future consideration. 🙂

**Reply by ellierider** [reply]
Thanks Kelsey!
